.github/workflows/lint-test.yml

```diff
@@ -31,11 +31,11 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Install Python Dependencies
-        uses: HassanAbouelela/actions/setup-python@setup-python_v1.3.1
+        uses: HassanAbouelela/actions/setup-python@setup-python_v1.4.0
         with:
           # Set dev=true to install flake8 extensions, which are dev dependencies
-          dev: true
-          python_version: '3.10'
+          python_version: '3.11'
```

Comment:
Actually, we might need to preserve the dev input we provide, I'll confim in a bit